modin/pandas/test/test_groupby.py

```diff
@@ -439,6 +439,14 @@ def maybe_get_columns(df, by):
     ):
         # Not yet supported for non-original-column-from-dataframe Series in by:
         eval___getattr__(modin_groupby, pandas_groupby, "col3")
+        eval__getitem__(modin_groupby, pandas_groupby, "col3")
+
+    non_by_cols = (
+        [col for col in pandas_df.columns[1:] if col not in modin_groupby._internal_by]
```

Comment:
do we have a separate test for `._internal_by` property?

Reply:
added a [test](https://github.com/dchigarev/modin/blob/75ae80d64ff790ef2e1d158e31e3c138517408ee/modin/pandas/test/test_groupby.py#L1516) for `._internal_by` at the last commit